### PR TITLE
changes to episode 1, tasks 3, 5, 6, 8, 10

### PR DESCRIPTION
**Pull request by mallewellyn**
Tasks 3, 5, 6, 8, 10 of issue #112 

To summarise:
- Minor changes to Challenges wording.
- Adding links between Challenges and main text.

### CODE REVIEW

File: _episodes_rmd/01-introduction-to-high-dimensional-data.Rmd [hunk 1]
```diff
@@ -65,7 +65,7 @@ specific patient outcomes (e.g. survival, length of time spent in hospital).
 An example of what high-dimensional data might look like in a biomedical study
 is shown in the figure below. 
 
-```{r table-intro, echo = FALSE}
+```{r table-intro, echo = FALSE, fig.cap = "Example of a high-dimensional data table with features in the columns and individual observations (patients) in rows.", fig.alt = "Table displaying a high-dimensional data set with many features in individual columns relating to health data such as blood pressure, heart rate and respiratory rate. Each row contains the data for individual patients."}
 knitr::include_graphics("../fig/intro-table.png")
```
Comment (alanocallaghan) [inline _episodes_rmd/01-introduction-to-high-dimensional-data.Rmd:69]:
```suggestion
knitr::include_graphics(here::here("fig/intro-table.png"))
```

File: _episodes_rmd/01-introduction-to-high-dimensional-data.Rmd [hunk 2]
```diff
@@ -195,7 +198,7 @@ dataset are almost equal. In that situation the effective number of observations
 per features is low. The result of fitting a best fit line through
 few observations can be seen in the right-hand panel below.
 
-```{r intro-figure, echo = FALSE}
+```{r intro-figure, echo = FALSE, fig.cap = "Scatter plot of two variables (x and y) from a data set with 25 observations (left) and 2 observations (right) with a fitted regression line (red).", fig.alt = "Two scatter plots side-by-side, each plotting the relationship between two variables. The scatter plot on the left hand side shows 25 observations and a regression line with the points evenly scattered around. The scatter plot on the right hand side shows 2 observations and a regression line that goes through both points."}
 knitr::include_graphics("../fig/intro-scatterplot.png")
 # ![Figure 2: Least squares regression using a) low-dimensional data and b) low ratio of observations per feature](D:/Statistical consultancy/Consultancy/Grant applications/UKRI teaching grant 2021/Working materials/Figure 2 for Intro.png)
```
Comment (alanocallaghan) [inline _episodes_rmd/01-introduction-to-high-dimensional-data.Rmd:203]:
```suggestion
knitr::include_graphics(here::here("fig/intro-scatterplot.png"))
```

Comment (alanocallaghan) [reply _episodes_rmd/01-introduction-to-high-dimensional-data.Rmd:203]: This should replace the previous line, not just the old figure link

File: _episodes_rmd/01-introduction-to-high-dimensional-data.Rmd [hunk 3]
```diff
@@ -210,14 +212,15 @@ common problem when using regression on high-dimensional datasets.
...
+explore why high correlations might be an issue in a Challenge.
 
 
 > ## Challenge 3
 > 
 > Use the `cor()` function to examine correlations between all variables in the 
-> `Prostate` dataset. Are some pairs of variables highly correlated (i.e. 
-> correlation coefficients > 0.6)?
+> `Prostate` dataset. Are some pairs of variables highly correlated using a threshold of 
+> 0.7 for the correlation coefficients?
```
Comment (alanocallaghan) [inline _episodes_rmd/01-introduction-to-high-dimensional-data.Rmd:223]:
```suggestion
> 0.75 for the correlation coefficients?
```

if we remove the index column from the prostate data, then this 0.75 threshold only identifies gleason <-> pgg45

File: _episodes_rmd/01-introduction-to-high-dimensional-data.Rmd [hunk 4]
```diff
@@ -274,8 +277,14 @@ in these datasets makes high correlations between variables more likely.
 
 Including highly correlated variables such as `gleason` and `pgg45` 
 simultaneously the same regression model can lead to problems 
-in fitting a regression model and interpreting its output. To allow variables to 
-be included in the same model despite high levels of correlation, we can use
+in fitting a regression model and interpreting its output. Although, each variable
```
Comment (alanocallaghan) [inline _episodes_rmd/01-introduction-to-high-dimensional-data.Rmd:280]:
```suggestion
in fitting a regression model and interpreting its output. Although each variable
```

**Review by alanocallaghan — APPROVED** [review]
This is great, really appreciate the attention to detail (spotting the use of "challenges" with respect to difficulties and exercises, the proper distinction between caption and alt text) and generally having a fresh pair of eyes

**Comment by mallewellyn** [comment]
Thanks Alan! I'm glad the suggestions were helpful!

**Pull request merged**